features/step_definitions/meta_tag_steps.rb

```diff
@@ -0,0 +1,29 @@
+Then(/^there should be meta tags for the edit profile form$/) do
+  expect(page).to have_css("title", visible: false, text: "18F Micro-purchase - Edit user: #{@user.name}")
+end
+
+Then(/^there should be meta tags for the closed auction$/) do
+  pr_auction = AuctionPresenter.new(@auction)
+
+  expect(page).to have_css("title", visible: false, text: "18F Micro-purchase - #{pr_auction.title}")
```

Comment:
this `title` is just delegated to auction -- thoughts on using `@auction.title` here?